StandaloneMmPkg/Library/StandaloneMmCoreEntryPoint/Arm/StandaloneMmCoreEntryPoint.c

```diff
@@ -541,56 +505,16 @@ ValidateMmCommBufferAddr (
   }
 
   // perform bounds check.
-  if ((CommBufferRange - sizeof (EFI_MM_COMMUNICATE_HEADER)) <
-      ((EFI_MM_COMMUNICATE_HEADER *)CommBufferAddr)->MessageLength)
+  if (((CommBufferAddr + sizeof (EFI_MM_COMMUNICATE_HEADER) +

```

Comment:
This looks like a bug fix that should have been part of one of the previous patches, can you check, please?

Reply:
No. What I remember is the functionality is the same, However Change the way to check boundary 
with renaming value. That's why it doesn't need to include previous patch.